Product Designer/assets/css/styles.css

```diff
@@ -0,0 +1,203 @@
+/* UNIVERSAL STYLES */
+*{
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    color: white;
+    padding: 0;
+    box-sizing: border-box;
+    margin: 0px;
+}
+body{
+    background-color: black;
+}
+
+/* MAIL SECTION STYLES*/
+.mailSection{
+    width: 74%;
+    margin: 0 auto;
+    display: flex;
+    justify-content: end;
+    font-size: 2.5em;
+    font-weight: 900;
+    height: 163px;
+    align-items: center;
+}
+.mailSection h6{
+    letter-spacing: 0.5px;
```

Comment:
Maintain consistent units of measurements, stick to either px or em , rem

Reply:
Yes changed all measurements  to px